docs/learn/terra-station/multisig.md

```diff
@@ -1,81 +1,80 @@
 # Multisig wallets
 
-Multisig wallets are an advanced feature of Terra Station. If you’re using Terra Station for the first time, follow the [Terra Station tutorial](download/terra-station-desktop.md).
+Multisig wallets are an advanced feature of Terra Station. If you’re using Terra Station for the first time, follow the [Terra Station tutorial](download/terra-station-desktop.md) to learn more.
 
 Multisig wallets enable a wallet to be controlled by multiple parties. A wallet manager creates a transaction and sends an encoded transaction message to the wallet signers. The signers sign the transaction and send back their signatures. The wallet manager then inputs the encoded transaction message along with the received signatures to complete the transaction.
 
 ## Prerequisites
 
-- Download the [Terra Station browser extension](download/terra-station-chrome.md)
+- Download the [Terra Station browser extension](download/terra-station-chrome.md).
 
 ## Create a multisig wallet
 
-1. Open the Terra Station browser extension and click **New multisig wallet**.
+1. Open the Terra Station browser extension, click on **Add a wallet**, and then click **New multisig wallet**.
 
-2. Enter the wallet addresses of each multisig user in the correct order.
+2. Enter the wallet addresses of each user of the multisig wallet.
+
+3. In the **Threshold** field, enter the number of signatures needed to post a transaction .
 
    :::{caution}
-   Each wallet added must have a transaction history of at least one transaction before it can be added to a multisig wallet.
+   You will need to record the order of the inputted wallet addresses as well as the threshold value for initial transaction confirmation.  Each wallet address provided must have a transaction history of at least one transaction before it can be added as a signer of a multisig wallet.
```

Comment:
Although both words are technically correct, "input" is more common than "inputted."

Reply:
@evanorti updated "the order of the **inputted wallet addresses**" to "the order of the **wallet addresses provided**".